Gruntfile.js

```diff
@@ -112,6 +112,7 @@ module.exports = function(grunt) {
             '../../js/dropdown.jquery': './webapp/node_modules/bootstrap/js/dropdown', // enketo currently duplicates bootstrap's dropdown code.  working to resolve this upstream https://github.com/enketo/enketo-core/issues/454
             'angular-translate-interpolation-messageformat': './webapp/node_modules/angular-translate/dist/angular-translate-interpolation-messageformat/angular-translate-interpolation-messageformat',
             'angular-translate-handler-log':  './webapp/node_modules/angular-translate/dist/angular-translate-handler-log/angular-translate-handler-log',
+            'moment': './webapp/node_modules/moment/moment'
```

Comment:
Should we do this for all libraries in shared-libs (eg: underscore)? Maybe we should enforce that shared libs don't have any dependencies, or have a test than ensures all dependencies are listed in this browserify alias map?

Reply:
> Maybe we should enforce that shared libs don't have any dependencies

Reply:
> Maybe we should enforce that shared libs don't have any dependencies, or have a test than ensures all dependencies are listed in this browserify alias map?

The alias map sounds like a legit thing. I wonder if it's possible to just generically have browserify complain if we're pulling in more than one version of the same library, because presumably we never want that to happen?

I wouldn't want to remove dependencies from shared libs entirely, then you couldn't isolate their tests etc, and I like that at least for now they are one publish away from being a legit npm package.

Reply:
Actually, I was wrong. There are 2 versions of `moment` browserified, `2.22.0` & `2.22.2`.
We use `2.22.2`, while `bootstrap-daterangepicker` has version `2.9.0` as a dependency - which yarn resolves to `2.22.0`.